src/leiningen/i18n/bin/update-pot.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+set -eo pipefail
+
+repo_dir="$(realpath "$(dirname "$0")/../../..")"
+pot_file="$repo_dir/locales/messages.pot"
```

Comment:
Yeah I think you're right. It could be package name, but given we'd be changing it everywhere - it's best to wait and not future proof for now.